src/Core/Scheduler/Scheduler.js

```diff
@@ -91,9 +92,9 @@ Scheduler.prototype.initDefaultProviders = function initDefaultProviders() {
     this.addProtocolProvider('tms', new TMS_Provider());
     this.addProtocolProvider('potreeconverter', PointCloudProvider);
     this.addProtocolProvider('wfs', new WFS_Provider());
+    this.addProtocolProvider('vectortocolor', new VectorToColor_Provider());
```

Comment:
What do you think of `rasterizer` instead of `vectortocolor`?

Reply:
yes